test/services/account/services/accounts.test.js

```diff
@@ -371,11 +512,37 @@ describe('Account Service', () => {
 				.get('/accounts')
 				.query({ username: { $gte: 0 } })
 				.end((response, err) => {
-					expect(err).to.have.status(400);
+					expect(err).to.have.status(401);
 					done();
 				});
 		});
 
+		it.skip('should return an error if populate is specified in query params for a FIND method', async () => {
```

Comment:
why is this test skipped?

Reply:
I didn't know how to test it because query.params are overwrited in restrictAccess method - src/services/account/hooks/index.js and it removes populate from the query. I have just deleted this test